base/error.go

```diff
@@ -45,6 +47,8 @@ func (e SGError) Error() string {
 		return "Import cancelled"
 	case importCasFailure:
 		return "CAS failure during import"
+	case importRevFailure:
+		return "CAS failure during Rev import"
```

Comment:
Unless this issue only happens during 'import' (as opposed to just any write), let's not use the term 'import' to avoid confusion.  I also don't think there's a CAS failure involved.